Model/Resolver/setRzpPaymentDetailsForOrder.php

```diff
@@ -0,0 +1,210 @@
+{
+    /**
+     * @var GetCartForUser
+     */
+    private $getCartForUser;
+
+    /**
+     * @var SetPaymentMethodOnCartModel
+     */
+    private $setPaymentMethodOnCart;
+
+    /**
+     * @var CheckCartCheckoutAllowance
+     */
+    private $checkCartCheckoutAllowance;
+
+    /**
+     * @var \Magento\Framework\App\ObjectManager
+     */
+    protected $_objectManager;
+
+    /**
+     * @var \Magento\Sales\Api\Data\OrderInterface
+     */
+    protected $order;
+
+    /**
+     * @var \Razorpay\Magento\Model\Config
+     */
+    protected $config;
+
+    /**
+     * @var \Magento\Sales\Model\Service\InvoiceService
+     */
+    protected $invoiceService;
+
+    /**
+     * @var \Magento\Framework\DB\Transaction
+     */
+    protected $transaction;
+
+    /**
+     * @var \Magento\Framework\App\Config\ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    /**
+     * @var STATUS_PROCESSING
+     */
+    protected const STATUS_PROCESSING = 'processing';
+
+    /**
+     * @param GetCartForUser $getCartForUser
+     * @param SetPaymentMethodOnCartModel $setPaymentMethodOnCart
+     * @param CheckCartCheckoutAllowance $checkCartCheckoutAllowance
+     * @param PaymentMethod $paymentMethod
+     * @param \Magento\Sales\Api\Data\OrderInterface $order
+     * @param \Razorpay\Magento\Model\Config $config
+     * @param \Magento\Sales\Model\Service\InvoiceService $invoiceService
+     * @param \Magento\Framework\DB\Transaction $transaction
+     * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+     */
+    public function __construct(
+        GetCartForUser $getCartForUser,
+        SetPaymentMethodOnCartModel $setPaymentMethodOnCart,
+        CheckCartCheckoutAllowance $checkCartCheckoutAllowance,
+        PaymentMethod $paymentMethod,
+        \Magento\Sales\Api\Data\OrderInterface $order,
+        \Razorpay\Magento\Model\Config $config,
+        \Magento\Sales\Model\Service\InvoiceService $invoiceService,
+        \Magento\Framework\DB\Transaction $transaction,
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+    ) {
+        $this->getCartForUser             = $getCartForUser;
+        $this->setPaymentMethodOnCart     = $setPaymentMethodOnCart;
+        $this->checkCartCheckoutAllowance = $checkCartCheckoutAllowance;
+        $this->rzp                        = $paymentMethod->rzp;
+        $this->order                      = $order;
+        $this->config                     = $config;
+        $this->invoiceService             = $invoiceService;
+        $this->transaction                = $transaction;
+        $this->scopeConfig                = $scopeConfig;
+        $this->_objectManager             = \Magento\Framework\App\ObjectManager::getInstance();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
+    {
+        if (empty($args['input']['order_id'])) {
```

Comment:
@enishant , can we follow the `{` must start with new line every where in the code.

Reply:
@ramth05 Updated code to begin `{` on new line every where in the code.